src/main.js

```diff
@@ -3,7 +3,7 @@ const SUBSCRIBE_ENDPOINT = 'https://api.treasurehacks.org/notifications/live-sit
 if ('serviceWorker' in navigator) {
   navigator.serviceWorker.register(new URL('/src/service-worker.js', import.meta.url)).then(sw => {
     if ('Notification' in window) {
-      if (Notification.permission === 'granted') {
+      if (Notification.permission !== 'granted') {
```

Comment:
Since this is inside of a function where you only care for `permission !== 'granted'`, it's better to use a guard clause

`if (permission === 'granted') return`
Then do the request logic with a layer less of nesting/indentation